docs/tutorials/excel-tutorial.md

```diff
@@ -469,7 +469,7 @@ When a table is long enough that a user must scroll to see some rows, the header
 
    - The `Worksheet.freezePanes` collection is a set of panes in the worksheet that are pinned, or frozen, in place when the worksheet is scrolled.
 
-   - The `freezeRows` method takes as a parameter the number of rows, from the top that are to be pinned in place. We pass `1` to pin the first row in place.
+   - The `freezeRows` method takes as a parameter the number of rows, from the top, that are to be pinned in place. We pass `1` to pin the first row in place.
```

Comment:
This is probably a separate issue, but "we" is typically avoided in our tutorials. I'll tag @ElizabethSamuel-MSFT to confirm.

Reply:
Agreed. I think updating the sentence to "Pass `1` to pin the first row in place." should work.

Reply:
Cool. Since there's a few more instances of "we" in this tutorial, let's make changing that a separate PR. Might be worth doing a search for "we " in the other tutorials too. 

BTW, "we" is okay in the context of a Microsoft recommendation (e.g., "We recommend using VS Code."

Reply:
Ok, thanks for the context! Happy to make a separate PR and remove "we" from the tutorial.